tutorials/simulation_exporting.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.            data =  displacements + jumps + pressures
</pre><p>Having porepy novices in mind. Although, I honestly very much like your proper demonstration of how to define different data, I wonder whether this last example should not only focus on the iterations. The exporting of displacement jumps has been covered e.g. in TailoredExporting. And there is not structural difference between exporting displacements and pressures. Would it make sense to reduce the exported amount of data in this example to allow the user to grasp the essence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I wholeheartedly agree with you but for two reasons:
1) It may be of interest to show how different data (primary/secondary variables) are fetched from the iterate dictionary.
2) This part of the tutorial is not intended for porepy novices.

I can be persuaded to change this, though.

Reply:
I am OK with leaving it.